src/plugins/crypto.c

```diff
@@ -959,14 +959,16 @@ gboolean bd_crypto_luks_resize (const gchar *luks_device, guint64 size, GError *
  * @pass_data: (array length=data_len): a passphrase for the TrueCrypt/VeraCrypt volume (may contain arbitrary binary data)
  * @data_len: length of the @pass_data buffer
  * @read_only: whether to open as read-only or not (meaning read-write)
+ * @keyfiles: (array length=keyfiles_count) paths to the keyfiles for the TrueCrypt/VeraCrypt volume
+ * @keyfiles_count: length of the @keyfiles array
```

Comment:
We do zero/NULL terminated arrays where possible.

Reply:
Fixed in 8f0f434.

Reply:
That is now done in commit 69a3223 after the rebasing.